config/grafana/operations_dashboard.go

```diff
@@ -0,0 +1,220 @@
+			Query(dashboard.StringOrMap{
+				String: cog.ToPtr("label_values(tanka_environment_info{app=\"cloudcost-exporter\"},exported_cluster)"),
+			}).
+			Datasource(prometheusDatasourceRef()).
+			Multi(true).
+			Refresh(dashboard.VariableRefreshOnDashboardLoad).
+			IncludeAll(true).
+			AllValue(".*"),
+		).
+		Annotation(dashboard.NewAnnotationQueryBuilder().
+			Name("Annotations & Alerts").
+			Datasource(dashboard.DataSourceRef{
+				Type: cog.ToPtr[string]("grafana"),
+				Uid:  cog.ToPtr[string]("-- Grafana --"),
+			}).
+			Hide(true).
+			IconColor("rgba(0, 211, 255, 1)").
+			Type("dashboard").
+			BuiltIn(1),
+		).
+		WithRow(dashboard.NewRowBuilder("Overview")).
+		WithPanel(collectorStatusCurrent().Height(6).Span(12)).
+		WithPanel(collectorScrapeDurationOverTime().Height(8).Span(24)).
+		WithRow(dashboard.NewRowBuilder("AWS")).
+		WithPanel(costExplorerAPIRequestsOverTime().Height(6).Span(12)).
+		WithPanel(awsS3NextPricingMapRefreshOverTime().Height(6).Span(12)).
+		WithRow(dashboard.NewRowBuilder("GCP")).
+		WithPanel(gcpListBucketsRPSOverTime().Height(6).Span(12)).
+		WithPanel(gcpNextScrapeOverTime().Height(6).Span(12))
+
+	sampleDashboard, err := builder.Build()
+	if err != nil {
+		panic(err)
+	}
+	dashboardJson, err := json.MarshalIndent(sampleDashboard, "", "  ")
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(string(dashboardJson))
+}
+
+func collectorScrapeDurationOverTime() *timeseries.PanelBuilder {
+	return timeseries.NewPanelBuilder().
+		Title("Collector Scrape Duration").
+		Description("Duration of scrapes by provider and collector").
+		Datasource(prometheusDatasourceRef()).
+		WithTarget(
+			prometheusQuery("cloudcost_exporter_collector_last_scrape_duration_seconds", "{{provider}}:{{collector}}"),
+		).
+		Unit("s").
+		ThresholdsStyle(common.NewGraphThresholdsStyleConfigBuilder().Mode(common.GraphThresholdsStyleModeLine)).
+		Thresholds(dashboard.NewThresholdsConfigBuilder().
+			Mode(dashboard.ThresholdsModeAbsolute).
+			Steps([]dashboard.Threshold{
+				{Color: "green"},
+				{Value: cog.ToPtr[float64](60), Color: "red"},
+			}),
+		).
+		ColorScheme(dashboard.NewFieldColorBuilder().Mode("palette-classic")).
+		Legend(common.NewVizLegendOptionsBuilder().
+			DisplayMode(common.LegendDisplayModeTable).
+			Placement(common.LegendPlacementBottom).
+			ShowLegend(true).
+			SortBy("Last *").
+			SortDesc(true).
+			Calcs([]string{"lastNotNull",
+				"min",
+				"max"}),
+		)
+}
+
+func costExplorerAPIRequestsOverTime() *timeseries.PanelBuilder {
+	return timeseries.NewPanelBuilder().
+		Title("CostExplorer API Requests").
+		Datasource(prometheusDatasourceRef()).
+		WithTarget(
+			prometheusQuery(
+				"sum by (cluster) (increase(cloudcost_exporter_aws_s3_cost_api_requests_total{cluster=~\"$cluster\"}[5m]))",
+				"__auto",
+			),
+		).
+		Unit("reqps").
+		Legend(common.NewVizLegendOptionsBuilder().
```

Comment:
All of these `Legend()` calls can most likely be removed: their configuration seems to be the default one

Reply:
Good call. I think started with the one panel that _needed_ the legend and then didn't experiment with the rest.